assets/js/googlesitekit/datastore/forms/index.js

```diff
@@ -82,10 +82,10 @@ export const selectors = {
 	 * @param {Object} state Data store's state.
 	 * @param {string} formName Name of the form.
 	 * @param {string} key Key supplied from formData.
-	 * @return {string|undefined} Value stored in state by formName and key or undefined.
+	 * @return {*} Value stored in state by formName and key or undefined.
 	 */
 	getValue( state, formName, key ) {
-		return state[ formName ][ key ];
+		return state[ formName ] ? state[ formName ][ key ] : undefined;
```

Comment:
I find this to be a bit easier to read, what do you think?
```suggestion
		const formData = state[ formName ] || {};

		return formData[ key ];
```